apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -169,6 +171,8 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         [ "data-testid" ]: testId
     } = props;
 
+    const [ subjectDN, setTLSClientAuthSubjectDN ] = useState(initialValues?.clientAuthentication ?
```

Comment:
Keep useState hooks grouped together for readability.

Reply:
fixed with [a5f0e39](https://github.com/wso2/identity-apps/pull/4284/commits/a5f0e399c3a9fa90a48a937face8fe7d2f554faf)